views/index.ejs

```diff
@@ -57,7 +57,15 @@
       </form>
       <div class="search_form__result"></div>
     </div>
-    <script>window.times = {init : Date.now()}</script>
+    <script>
+        window.times = {init : Date.now()}
+
+        // iframe: hide UI immediately to avoid content flashing while the JS bundle loads
+        if (location.search.includes('ui=0')) {
+            window.no_ui = true;
```

Comment:
maybe we should keep the same name consistent with the url query param ?
`window.ui = false`